packages/zoe/test/unitTests/test-instanceStorage.js

```diff
@@ -121,7 +128,8 @@ test('makeInstanceRecordStorage', async t => {
   // Add currency again, but call it "money"
   addIssuerToInstanceRecord(
     'Money',
-    makeIssuerRecord(currencyKit.brand, currencyKit.issuer, AssetKind.NAT, {
+    makeIssuerRecord(currencyKit.brand, currencyKit.issuer, {
```

Comment:
The brace here moved, too.  I think the braces should surround `decimalPlaces` to make a `displayInfo`, and assetKind should be a separate third argument.

Reply:
Why do you think so? The types were telling me that `assetKind` should be a part of `displayInfo`, which is the third argument.

Reply:
Never mind. I was reading the body of `makeIssuerRecord()` (in `issuerRecord.js`) and thought I was looking at the parameter list.

I was only looking at the def'n because `AssetKind.NAT` moved from being the third param to being a part of the third param.  Was the code previously broken? Why didn't `tsc` catch it before?

Reply:
`tsc` doesn't cover the tests yet in Zoe